docs/src/main/sphinx/connector/opensearch.md

```diff
@@ -83,6 +83,9 @@ The following table details all general configuration properties:
     queries. Some deployments map OpenSearch ports to a random public port and
     enabling this property can help in these cases.
   - `false`
+* - `opensearch.projection-pushdown-enabled`
```

Comment:
What's the motivation for making this configurable?

Reply:
It's ok to keep it as a fallback while we test out this feature, but we should have a plan to remove it in a few releases.